types/i18n.d.ts

```diff
@@ -72,6 +72,10 @@ declare global {
 
     /** Localized strings for the html report renderer. */
     export type I18NRendererStrings = typeof Util['UIStrings'];
+
+    export type LhlMessages = Record<string, {message: string}>;
+
+    export type Strings = Record<LH.Locale, LhlMessages>;
```

Comment:
I'd really like to avoid making global types when they aren't needed, or at least scope these to `treemap.d.ts`. Defining `LhlMessages` in `locales.js` where the actual object is also defined was very intentional, and `LH.Strings` is a simple and can just be defined in place.

Reply:
removed Strings. but LhlMessages is used in 6 different files, so despite intention it is being used globally just in the more cumbersome jsdoc import style.

Reply:
> so despite intention it is being used globally just in the more cumbersome jsdoc import style.

Strong disagree. At no point has anyone suggested setting `global.i18n = require('./lighthouse-core/lib/i18n/i18n.js')` so we can avoid `require('../../../lib/i18n/i18n.js')` in all the files that use it, for instance.

I think that it's not worth hashing out the usefulness of globals (whether in value- or type-space) in this PR, though, but more importantly I think there's enough going on in here without making changes to types in core. Can we revert and follow up later? Happy to discuss at another time.